File: java/t/locale_speed.java
import com.iafan.plurr.*;

class locale_speed {
  public static void main(String[] args) throws PlurrLocaleNotFoundException {
    Plurr p = new Plurr();
    int x = 1000000;

    long start = System.nanoTime();

    for (int i = 0; i < x; i++) {
      p.setLocale("ru");
    }

    long end = System.nanoTime();
    double time = new Long((end - start) / 1000000).doubleValue();
    System.out.printf("Execution time (%d calls): %f sec (%f ms per call)", x, time / 1000, time / x);
  }
}

File: python/t/locale_speed.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import sys
sys.path.append("..")

import time

from Plurr import Plurr

p = Plurr()
x = 100000

start = time.clock()

for i in xrange(x):
  dummy = p.locale('ru')

end = time.clock()
time = end - start
print('Execution time ({0} calls): {1} sec ({2} ms per call)'.format(x, time, time * 1000 / x))
